File: leetcode-submissions/179_Largest_Number.cpp
class Solution {
public:

//creating custom comaprator
static bool customcmp(const int &a,const int &b){
    return to_string(a)+to_string(b)>to_string(b)+to_string(a);
}
    string largestNumber(vector<int>& nums) {
        sort(nums.begin(),nums.end(),customcmp);
        string result;
        for(int ele:nums){
            result+=to_string(ele);
        }
        return result;
    }
};

File: leetcode-submissions/179_Largest_Number.ts
function largestNumber(nums: number[]): string {
    nums.sort((a,b)=>(b.toString()+a.toString()).localeCompare(a.toString()+b.toString()) );
    let ans:string="";
    for(const ele of nums){
        ans+=ele.toString();
    }
    return ans.charAt(0)==="0"?"0":ans;
};